`cargo build --all-features` fails

Seems there are some leftovers in the code:

```
error[E0425]: cannot find value `ccc` in this scope
   --> rawler/src/ljpeg92.rs:278:33
    |
278 |     inspector!("ITER SIZE: {}", ccc);
    |                                 ^^^ not found in this scope

error[E0283]: type annotations needed
   --> rawler/src/decoders/iiq.rs:198:44
    |
198 |       1 if tag == IiqTag::SensorCorrection.into() => {
    |                --                          ^^^^
    |                |
    |                type must be known at this point
    |
    = note: multiple `impl`s satisfying `u32: PartialEq<_>` found in the following crates: `core`, `serde_json`:
            - impl PartialEq for u32;
            - impl PartialEq<serde_json::Value> for u32;
help: try using a fully qualified path to specify the expected types
    |
198 |       1 if tag == <IiqTag as Into<T>>::into(IiqTag::SensorCorrection) => {
    |                   ++++++++++++++++++++++++++                        ~

error[E0599]: no method named `components` found for mutable reference `&mut LjpegCompressor<'a>` in the current scope
   --> rawler/src/ljpeg92.rs:680:22
    |
680 |     for comp in self.components() {
    |                      ^^^^^^^^^^-- help: remove the arguments
    |                      |
    |                      field, not a method

Some errors have detailed explanations: E0283, E0425, E0599.
For more information about an error, try `rustc --explain E0283`.
error: could not compile `rawler` (lib) due to 4 previous errors
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -25,6 +25,21 @@ jobs:
       - run: cargo test
       - run: cargo check
 
+  build-all-features:
+    name: Rust all-features ${{matrix.rust}}
+    runs-on: ubuntu-20.04
+    strategy: 
+      fail-fast: false
+      matrix: 
+        rust: [stable]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{matrix.rust}}
+      - run: cargo build --all-features
+      - run: cargo check --all-features
+
   samplecheck:
     name: Testdata validation with Rust ${{matrix.rust}}
     runs-on: testdata-avail

--- a/rawler/src/ljpeg92.rs
+++ b/rawler/src/ljpeg92.rs
@@ -275,7 +275,6 @@ impl HuffTableBuilder {
         }
       }
     }
-    inspector!("ITER SIZE: {}", ccc);
     #[cfg(feature = "inspector")]
     for (i, codesize) in self.codesize.iter().enumerate() {
       inspector!("codesize[{}]={}", i, codesize);
@@ -677,7 +676,7 @@ impl<'a> LjpegCompressor<'a> {
     }
 
     #[cfg(feature = "inspector")]
-    for comp in self.components() {
+    for comp in 0..self.components {
       inspector!("Huffman table {}", comp);
       for i in 0..17 {
         inspector!("scan: self.huffman[{}].hist[{}]={}", comp, i, self.comp_state[comp].histogram[i]);

